src/main/java/edu/uc/cech/agrawadv/eventorganizer/user/User.java

```diff
@@ -10,35 +10,40 @@
 @Entity
 @Table(name = "users")
 public class User {
+	
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	@Column(name = "userid")
 	private int userId;
+	
 	@Column(name = "name")
 	private String name;
+	
 	@Column(name = "username")
 	private String username;
+	
 	@Column(name = "password")
 	private String password;
+	
 	@Column(name = "role")
-	private String role;
+	private String userRole;

```

Comment:
I renamed this to user role because it is the role for a user.
ref: 
https://dev.to/khalalw/the-5-commandments-of-naming-variables-1l04